static/webpack-config.js

```diff
@@ -66,6 +67,14 @@ module.exports = function () {
     })
   ];
 
+  const analyzeBundle = (process.env || {}).ANALYZE_BUNDLE === 'true';
+
+  if (analyzeBundle) {
+    plugins.push(new BundleAnalyzerPlugin({
+      analyzerMode: 'static',
```

Comment:
oh shoot I missed this the first time around, but did you say that the server mode is better? (I've only tried the static mode)

Reply:
I'm pretty sure the server mode just serves the same HTML page that you get with the static. I was having some issues with the server mode but static worked for me